src/components/CircularProgress/CircularProgress.js

```diff
@@ -81,14 +99,14 @@ const ProgressWrapper = styled.div`
 `;
 
 /**
- * Size is set in px
+ * Size "small" or "medium"
  * ```javascript
  * import React from 'react';
  * import { CircularProgress } from 'weaveworks-ui-components'
  *
  * React.render(
  *   <div>
- *     <CircularProgress size={35} />
+ *     <CircularProgress size={"small"} />
```

Comment:
And we don't need the `{}` wrapping the `""` either! But this is not a blocker.

If you wanna merge this I can bump the version 👌